infra/app/app-config/env-config/identity-provider.tf

```diff
@@ -15,7 +15,10 @@ locals {
   logout_url_path = ""
 
   identity_provider_config = var.enable_identity_provider ? {
-    identity_provider_name = "${local.prefix}${var.app_name}-${var.environment}"
+    # We don't prefix the identity provider name with the terraform workspace because
+    # non-default workspaces use the default workspace identity provider instead of
+    # creating a new resource.
+    identity_provider_name = "${var.app_name}-${var.environment}"
```

Comment:
@lorenyu I think we have to remove the prefix if we want to re-use existing idps across terraform workspaces. I think this is fine, but wanted to call it out in case you think there's a particular reason we shouldn't take this approach.

Reply:
This is right. Remember we actually have a backlog ticket to get rid of prefix from the config module altogether: https://github.com/navapbc/template-infra/issues/654

Reply:
Ah yes! Thank you for the reminder.